### ISSUE TITLE
Ability to go straight into custom css in the customizer?

### ISSUE THREAD

**Issue by wormeyman** [rosswintle/turbo-admin#27]
The shortcut link is `/wp-admin/customize.php?autofocus%5Bsection%5D=custom_css`

**Comment by wormeyman** [comment]
Here is the bookmarklet that i use:

`javascript:window.location.href=window.location.origin+'/wp-admin/customize.php?autofocus%5Bsection%5D=custom_css';
`

**Comment by rosswintle** [comment]
Hi Eric,

Thanks for this. I can see how this is useful, but in the context of Turbo Admin I need to consider it a bit more carefully:

- Is this link always present/working in WordPress? If not then what are the conditions for it being present?
- If I add this, do I then need to add a bunch of other shortcuts to things in the customizer?
- If so then I'll need to find a way to "scrape" the customizer options somehow.

Leave it with me and I'll see what I can do. I love the idea of being able to go straight to sections of the Customizer interface, but hopefully you can see that this is a bit more complex than just adding a link to Turbo Admin.

**Comment by wormeyman** [comment]
Yes I totally understand the need to not accept every feature request. Custom CSS is almost always present because a theme can turn it off in functions.php but I have never seen it.
